tests/test_user_signin.py

```diff
@@ -0,0 +1,58 @@
+import json, bcrypt
+
+from django.test  import Client
+from django.test  import TestCase
+
+from user.models  import User, AccountType
+from user.utils      import SIGNIN_USER
+
+
+class SignInTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        AccountType(name='email').save()
+        AccountType(name='kakao').save()
+
+        password = '12345qwert!@#$%'
+        hashed_pw = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
+
+        email_account_type = AccountType.objects.get(name="email")
+
+        User(
+            email        = 'homer@homer.com',
+            nickname     = 'King God Homer',
+            password     = hashed_pw,
+            phone_number = '01012345678',
+            account_type = email_account_type
+        ).save()
+
+
+    def test_signin_success(self):
+        client = Client()
+        response = client.post('/user/signin', json.dumps(SIGNIN_USER), content_type='application/json')
```

Comment:
`SIGNIN_USER` 데이터는 어떤 데이터 인가요?
```suggestion
        response = client.post('/user/signin', json.dumps(SIGNIN_USER), content_type='application/json')
```

Reply:
반복되는 코드라서 test 밖의 파일에 빼 두었던 변수입니다.
setUp에 담아서 불러오도록 수정하였습니다.